README.md

```diff
@@ -35,7 +35,18 @@ Generate orders with a specific status.
 
 You may wish to disable emails if creating a large number of orders as this will trigger emails. To block all emails from your site, install a plugin like [Disable Emails](https://wordpress.org/plugins/disable-emails/).
```

Comment:
This is out of date. Sending emails was disabled in #58. We should remove this paragraph.

Reply:
177bcc9